doc/example/patterns_retrieve_cref.cpp

```diff
@@ -40,11 +51,12 @@ namespace
 
 BOOST_AUTO_TEST_CASE( method_is_called_two_times_with_the_same_value )
 {
+    using namespace mock_test;
```

Comment:
Do we need this? It’s maybe better if we can keep all the code examples minimal and they all look the same.
It seems there is sometimes a namespace and sometimes not.

Reply:
I moved the test into the namespace so this is gone now